modules/ipi-install-replacing-a-bare-metal-control-plane-node.adoc

```diff
@@ -15,6 +15,17 @@ If you reuse the `BareMetalHost` object definition from an existing control plan
 Existing control plane `BareMetalHost` objects may have the `externallyProvisioned` flag set to `true` if they were provisioned by the {product-title} installation program.
 ====
 
+.Prerequisites
+
+* You have access to the cluster as a user with the cluster-admin role.
+
+* You have taken an etcd backup.
```

Comment:
Can you add link to the steps to take etcd backup? It should be in replacing unhealthy etcd member?

Reply:
I added a link to the etcd backup procedure in the additional resources section, see **installing/installing_bare_metal_ipi/ipi-install-expanding-the-cluster.adoc**. Our style guide does not allow us to add inline links, so I put the link in the additional resources section.

Reply:
okay, additional resources is good, can we mention like please refer to additional resources?